src/components/NavbarLink/index.js

```diff
@@ -1,20 +1,27 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import classnames from 'classnames'
 
-import './styles.scss'
+import classes from './styles.module.scss'
+
+const NavbarLink = ({ text, iconSrc, isCart, onClick }) => {
+  const linkClasses = classnames(classes.link, {
+    [classes.icon]: iconSrc,
+    [classes.cartIcon]: isCart
+  })
 
-const NavbarLink = ({ text, iconSrc, onClick }) => {
   return (
-      <button onClick={onClick}>
+      <button className={linkClasses} onClick={onClick}>
           {text}
-          {iconSrc ? <img src={iconSrc} alt="Icono" /> : ''}
+          {iconSrc ? <i className={iconSrc} /> : null}
```

Comment:
You can do `{ iconSrc && <i className={iconSrc} />}` (conditional rendering only with AND condition)